tsunami/prompts/system.md

```
@@ -0,0 +1,1365 @@
+
+- ✅ Read with atom.Get() in render code
+- ❌ Never call atom.Set() in render code - only in handlers/effects
+- ✅ Always use SetFn() for concurrent updates from goroutines
+
+## Tsunami App Template
+
+```go
+package main
+
+import (
+	_ "embed"
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)